unpackaged/test_managed_only/preferred_email/classes/CON_Pref_Email_untranslatedSpsh_MTST.cls

```diff
@@ -0,0 +1,348 @@
+                
+                Contact contactToBeInserted = new Contact();
+                contactToBeInserted.put('Lastname', 'Email ' + i);
+                contactToBeInserted.put('FirstName', 'Preferred');
+                contactToBeInserted.put('Description', i + ' Contact Description ' + i );
+                contactToBeInserted.put('Department', i + ' Dept Num: ' + i);
+                contactToBeInserted.put('hed__preferred_Email__c', 'Correo electrónico sin traducir');
+                
+                contactsToBeInsertedList.add(contactToBeInserted);
+            }
+
+                            
+            Database.DMLOptions dml = new Database.DMLOptions();
+            dml.DuplicateRuleHeader.AllowSave = true;
+            
+            Test.startTest();
+                List<Database.saveResult> results = Database.insert(contactsToBeInsertedList, dml);
+            Test.stopTest();
+            
+            for (Database.SaveResult sr : results) {
+                if (sr.isSuccess() == false) {
+                    System.assertEquals(label.PrefEmailMatchNotNull, sr.getErrors()[0].getMessage(), 'The Contact record should not be inserted.');
+                } else {
+                    System.assert(false, 'No record should be inserted as the field in Preferred Email is empty.');
+                }
+            }
+        }
+    }
+
+    /*****************************************************************************************************************************
+    * @description Tests inserting a Contact with Custom Email as Preferred Email and and an EDA Email field
+    *  in Spanish Language  will throw PreferredEmailRequiredError Error when Preferred Email Enforcement is Enabled.
+    *****************************************************************************************************************************/
+    @isTest
+    private static void populateCustomEmailAndAlternateEmailOnInsertSpanish() {
+
+        System.runAs(spanishUser) {
+            CON_Pref_Email_Utility_MTST.populateCustomEmailAndAlternateEmailOnInsert('Untranslated_Email__c', 'test@sf.org');
+        }    
+    }
+
+    /*************************************************************************************************************************************
+    * @description Tests updating a Contact with a Custom Email field(Untranslated Email) in Spanish Language updates Preferred Email and
+    * standard Email with the custom email's label and value respectively when Preferred Email Enforcement is Enabled.
+    **************************************************************************************************************************************/
+    @isTest
+    private static void populateCustomEmailOnUpdateSpanish() {
+        
+        System.runAs(spanishUser) {
+            hed.UTIL_CustomSettings_API.getSettingsForTests(
+                new hed__Hierarchy_Settings__c(hed__Disable_Preferred_Email_Enforcement__c = false)
```

Comment:
```suggestion
                new %%%NAMESPACE%%%Hierarchy_Settings__c(%%%NAMESPACE%%%Disable_Preferred_Email_Enforcement__c = false)
```

Reply:
Thanks for catching this one @jofsky. I have updated all the files now.